scrapers/ar/bills.py

```diff
@@ -326,6 +326,8 @@ def scrape_bill_page(self, bill):
                 media_type="application/pdf",
             )
 
+        yield bill
```

Comment:
If we're only supposed to be returning one item from this function, let's just do a regular `return`. `yield` has specific semantics that can be messy when you only need one object.